### ISSUE TITLE
[BUG] bokeh.models.Tooltip causes rendering error

### ISSUE THREAD

**Issue by jnettels** [bokeh/bokeh#12951]
### Software versions

Python version      :  3.11.0 | packaged by conda-forge | (main, Oct 25 2022, 06:12:32) [MSC v.1929 64 bit (AMD64)]
IPython version     :  (not installed)
Tornado version     :  6.2
Bokeh version       :  3.0.3, 3.1.0
BokehJS static path :  C:\Users\Nettelstroth\anaconda3\envs\bokeh3\Lib\site-packages\bokeh\server\static
node.js version     :  (not installed)
npm version         :  (not installed)
Operating system    :  Windows-10-10.0.22000-SP0

### Browser name and version

Chrome, Firefox, Edge

### Expected behavior

Show description tooltip on a ``Select`` widget, as described here: https://docs.bokeh.org/en/latest/docs/user_guide/interaction/tooltips.html#tooltips

### Observed behavior

When creating a html-file with ``python my-script.py``, everything works as excepted.
When creating a server session with ``bokeh serve``, the browser console prints the error below.

Update: Using a Tooltip on a ``HelpButton`` always works. Maybe this issue is specific to the ``Select`` widget?

### Example code

```Python
from bokeh.models.widgets import Select
from bokeh.layouts import row
from bokeh.plotting import show
from bokeh.io import curdoc
from bokeh.models import Tooltip, HelpButton
from bokeh.models.dom import HTML

tooltip = Tooltip(content=HTML("<b>HTML</b> also does not work"), position="right")
# tooltip = Tooltip(content="This does not work", position="right")
tooltip_str = 'This works'

dropdown = Select(title="A dropdown", value='Test', options=['Test'],
                  sizing_mode='scale_width',
                  description=tooltip_str,  # does not work with Tooltip
                  )
help_button = HelpButton(tooltip=tooltip)  # Always works

# show(dropdown)  # This always works
curdoc().add_root(row(dropdown, help_button))  # This only works with String descriptions
```

### Stack traceback or browser console output
```
Error rendering Bokeh items: Error: [object HTMLDivElement] is not serializable
    at d._encode (bokeh.min.js?v=d4a1eba0f969a1b9ce0d43946c7cc357f701ace7f693de6e710ab393d5c546733a7b463f96f9d25ba678b51e1993b83937846dd0c3247d5a7225ea09a8aa7879:196:2121)
    at d.encode (bokeh.min.js?v=d4a1eba0f969a1b9ce0d43946c7cc357f701ace7f693de6e710ab393d5c546733a7b463f96f9d25ba678b51e1993b83937846dd0c3247d5a7225ea09a8aa7879:196:937)
    at [serialize] (bokeh.min.js?v=d4a1eba0f969a1b9ce0d43946c7cc357f701ace7f693de6e710ab393d5c546733a7b463f96f9d25ba678b51e1993b83937846dd0c3247d5a7225ea09a8aa7879:204:1183)
    at d._encode (bokeh.min.js?v=d4a1eba0f969a1b9ce0d43946c7cc357f701ace7f693de6e710ab393d5c546733a7b463f96f9d25ba678b51e1993b83937846dd0c3247d5a7225ea09a8aa7879:196:1075)
    at d.encode (bokeh.min.js?v=d4a1eba0f969a1b9ce0d43946c7cc357f701ace7f693de6e710ab393d5c546733a7b463f96f9d25ba678b51e1993b83937846dd0c3247d5a7225ea09a8aa7879:196:937)
    at d._encode (bokeh.min.js?v=d4a1eba0f969a1b9ce0d43946c7cc357f701ace7f693de6e710ab393d5c546733a7b463f96f9d25ba678b51e1993b83937846dd0c3247d5a7225ea09a8aa7879:196:1179)
    at d.encode (bokeh.min.js?v=d4a1eba0f969a1b9ce0d43946c7cc357f701ace7f693de6e710ab393d5c546733a7b463f96f9d25ba678b51e1993b83937846dd0c3247d5a7225ea09a8aa7879:196:937)
    at S.create_json_patch (bokeh.min.js?v=d4a1eba0f969a1b9ce0d43946c7cc357f701ace7f693de6e710ab393d5c546733a7b463f96f9d25ba678b51e1993b83937846dd0c3247d5a7225ea09a8aa7879:165:6805)
    at a._document_changed (bokeh.min.js?v=d4a1eba0f969a1b9ce0d43946c7cc357f701ace7f693de6e710ab393d5c546733a7b463f96f9d25ba678b51e1993b83937846dd0c3247d5a7225ea09a8aa7879:229:967)
    at _document_listener (bokeh.min.js?v=d4a1eba0f969a1b9ce0d43946c7cc357f701ace7f693de6e710ab393d5c546733a7b463f96f9d25ba678b51e1993b83937846dd0c3247d5a7225ea09a8aa7879:229:122)
```

**Comment by jnettels** [comment]
I don't know if this helps, but: Using a Tooltip on a ``HelpButton`` always works. Maybe this issue is specific to the ``Select`` widget?

**Comment by mattpap** [comment]
It's specific to input widgets. It's fixed in PR #12952.
